sos/collector/clusters/juju.py

```diff
@@ -0,0 +1,209 @@
+# Copyright (c) 2023 Canonical Ltd., Chi Wai Chan <chiwai.chan@canonical.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import json
+import re
+
+from sos.collector.clusters import Cluster
+
+
+def _parse_option_string(strings=None):
+    """Parse commad separated string."""
```

Comment:
Nitpick: typo. Should be `comma`, correct?

Reply:
updated